### ISSUE TITLE
Pages should not display dark text in dark mode

### ISSUE THREAD

**Issue by cdw9** [boostorg/website-v2#1473]
Examples:
* https://www.stage.boost.cppalliance.org/doc/libs/release/libs/assign/doc/index.html
* https://www.stage.boost.cppalliance.org/doc/libs/release/libs/compatibility/index.html

![Image](https://github.com/user-attachments/assets/88c6d605-068c-44df-977f-4995f4aac80c)

**Comment by rbbeeston** [comment]
No, It needs to be addressed, even if it ends up being fixed as part of another ticket, it should be linked to this one before being closed out.

**Comment by rbbeeston** [comment]
It should have boostlook applied to it, that should fix it. @julioest can you confirm?

**Comment by julioest** [comment]
This library does **not** have the boostlook class applied.

**Comment by rbbeeston** [comment]
@julioest yes, I know it doesn't but I believe it should have it.  What needs to happen to add it?

**Comment by julioest** [comment]
@rbbeeston The logic responsible for wrapping the doc elements needs to be modified still. This is tied to Dave's work of identifying doc types.

**Comment by rbbeeston** [comment]
this was fixed when we made darkmode just for boostlook docs
